src/conf.c

```diff
@@ -254,7 +254,14 @@ conf_init(conf_t *conf)
 
 	if ((s2 = getenv("HOME")) != NULL) {
 		char s[MAX_STRING];
-		sprintf(s, "%s/%s", s2, ".axelrc");
+		int ret;
+
+		ret = snprintf(s, MAX_STRING, "%s/.axelrc", s2);
+		if (ret < (strlen(s2) + strlen("/.axelrc") + 1)) {
```

Comment:
The check should be for `ret >= sizeof(s)`, `snprintf` always returns the total, even if the output was truncated.

Reply:
of, the manpage fooled me. First it says one thing, then it corrects itself :P